src/py42/sdk/__init__.py

```diff
@@ -21,12 +47,18 @@ def from_local_account(host_address, username, password):
     Returns:
         :class:`py42.sdk.SDKClient`
     """
-    return SDKClient.from_local_account(host_address, username, password)
+    client = SDKClient.from_local_account(host_address, username, password)
+
+    # test credentials
+    client.users.get_current()
```

Comment:
Part of doesn't like that we are testing the connection here... I know we were before, but there have been a few circumstances where I wanted to test the connection outside of creating the SDK so I can handle the error differently.  Think of an integration's perspective where we are using py42 in a different application.

Reply:
worth discussing for sure, but as part of this refactor i think we should just aim to preserve existing behavior